packages/superset-ui-encodable/src/fillers/fillMissingPropertiesInChannelDef.ts

```diff
@@ -0,0 +1,32 @@
+import { ChannelDef } from '../types/ChannelDef';
+import { ChannelType } from '../types/Channel';
+import { isFieldDef } from '../typeGuards/ChannelDef';
+import fillAxisConfig, { FilledAxisConfig } from './fillAxisConfig';
+import fillScaleConfig, { FilledScaleConfig } from './fillScaleConfig';
+import { Value } from '../types/VegaLite';
+
+type FilledChannelDef<Output extends Value = Value> = Omit<
+  ChannelDef,
+  'title' | 'axis' | 'scale'
+> & {
+  axis: FilledAxisConfig;
+  scale: FilledScaleConfig<Output>;
+  title: string;
+};
+
+export default function fillMissingPropertiesInChannelDef<Output extends Value = Value>(
```

Comment:
just noting a 3rd type of naming, I think it'd be great to align on one convention for the "fillers"